requirements-server.txt

```diff
@@ -0,0 +1,15 @@
+# Dependencies for notebook server and interaction with JupyterHub
+jupyterhub==1.4.0
+jupyterlab==3.0.14
+notebook==6.3.0
+# Dependencies for jupyter widgets
+bqplot==0.12.25
+ipytree==0.1.8
+ipywidgets-extended==1.0.5
+nglview==2.7.7
+widget-periodictable==2.1.5
+# nbserverproxy. ### are we using this?
+nbserverproxy==0.8.8
+# Install voila package and AiiDAlab voila template. ### another candidate for removal? is this documented?
```

Comment:
what about voila - is this being used?

Reply:
I think we haven't made a final decision on whether to use it or not. Most probably, it will be used in Jupyter Lab. So for the moment, I would leave it as is.